index.html

```diff
@@ -2930,6 +2931,40 @@ <h2 id="View">Backbone.View</h2>
       events that the view has <a href="#Events-listenTo">listenTo</a>'d.
     </p>
 
+    <p id="View-events">
+      <b class="header">events</b><code>view.events or view.events()</code>
+      <br />
+      The <b>events</b> hash (or function) can be used to specify DOM event listeners
+      for your View. By explicitly defining the <b>events</b> property of your View,
+      Backbone will automatically attach the event listeners at instantiation time,
+      since it attempts to <a href="#View-delegateEvents">delegateEvents</a> right
```

Comment:
This line are unnecessary. How about "Backbone will automatically attach the event listeners at instantiation time, right before invoking initialize", and move the `delegateEvents` reference earlier.